Cargo.toml

```diff
@@ -58,7 +58,8 @@ regex = "1.7.1"
 runas = "1.0.0"
 rust-ini = "0.18.0"
 semver = "1.0.16"
-sentry = { version = "0.31.2", default-features = false, features = [
+# TODO: Revert back to crates.io version prior to merge
+sentry = {git = "https://github.com/getsentry/sentry-rust.git", rev = "refs/pull/625/head", default-features = false, features = [
```

Comment:
I have pinned here my pull request on `sentry-rust`, which adds the crontab validation, so that we can test this PR before the `sentry-rust` changes are merged.

We need to merge the `sentry-rust` PR before merging this PR and release a new `sentry-rust` version, and we should pin the new version here before merging.